### PR TITLE
Feature/doc edits

### PR DESCRIPTION
**Pull request by WilliamBerryiii**
Updates for language and edits for reader clarity.

### CODE REVIEW

**Review by katriendg — APPROVED** [review]
Awesome. Thank you so much for the edits and corrections!

File: deployment/remove-dev-resources.ps1 [hunk 1]
```diff
@@ -53,9 +52,9 @@ az ad sp delete --id $aksServicePrincipal.appId
 # $l3ResourceGroup = $ApplicationName + "L3"
 # $l2ResourceGroup = $ApplicationName + "L2"
 
-# # Remove App resource group
```
Comment (suneetnangia) [inline deployment/remove-dev-resources.ps1:56]: Should we update this to have "resource group" as "Resource Group", give it's a service name.

File: README.md [hunk 2]
```diff
@@ -33,7 +33,8 @@ This section describes the nested topology design implemented by this solution.
 ![alt text](architecture/nested-topology-hld.png "Nested Toplogy")
 
 At the core of the nested topology design, we have proxies (currently Squid) which broker the connections between each hypothetical ISA-95 level (Level 2,3,4 in this instance).
-These proxies prevent workloads and Arc agents running at lower levels from connecting to the outside world directly, allowing the traffic to be managed/controlled via proxy configuration at each level. We are currently enhancing this functionality by enforcing data plane to traverse via proxy as well, as we only have Arc configured for it for now.
+These proxies prevent workloads and Arc agents running at lower levels from connecting to the outside world directly, allowing the traffic to be managed/controlled via proxy configuration at each level.
+We are currently implementing functionality that will force the data plane to traverse via the proxy as well, but only Arc is currently configured.
```
Comment (suneetnangia) [inline README.md:37]: @katriendg thinking if we are still unsure re use of proxy for data plane?

**Review by suneetnangia — COMMENTED** [review]
Thank you @WilliamBerryiii, added a couple of comments for your review before we merge.

File: README.md (hunk 2 above)
Comment (katriendg) [reply README.md:37]: Also reflecting on this paragraph, we could also mention we are working on ensuring Container Registry downloads will be proxied too. This is a task on our board to review usage of Connected Container Registry, today it's still going directly to the public GH endpoints (or ACR when deploying the developer version).

File: README.md [hunk 3]
```diff
@@ -33,7 +33,8 @@ This section describes the nested topology design implemented by this solution.
 ![alt text](architecture/nested-topology-hld.png "Nested Toplogy")
 
 At the core of the nested topology design, we have proxies (currently Squid) which broker the connections between each hypothetical ISA-95 level (Level 2,3,4 in this instance).
-These proxies prevent workloads and Arc agents running at lower levels from connecting to the outside world directly, allowing the traffic to be managed/controlled via proxy configuration at each level. We are currently enhancing this functionality by enforcing data plane to traverse via proxy as well, as we only have Arc configured for it for now.
+These proxies prevent workloads and Arc agents running at lower levels from connecting to the outside world directly, allowing the traffic to be managed/controlled via proxy configuration at each level.
```
Comment (WilliamBerryiii) [inline README.md:36]:
```suggestion
These proxies prevent workloads and Arc agents running at lower levels from connecting to the outside world directly, allowing the traffic to be managed or controlled via proxy configuration at each level. We are currently implementing functionality for data planes to transverse layers directly and evaluating an improvement to force this communication to pass through the proxy transparently. Currently only Azure Arc is configured to connect through proxy.
```

File: README.md (hunk 2 above)
Comment (WilliamBerryiii) [reply README.md:37]:
```suggestion
```

**Comment by WilliamBerryiii** [comment]
@suneetnangia & @katriendg please re-review based on feedback changes.

**Review by katriendg — DISMISSED** [review]
Thanks! Lgtm

**Review by suneetnangia — DISMISSED** [review]
Looks good, thanks @WilliamBerryiii

**Comment by katriendg** [comment]
Merged back from main which had some conflicts. @WilliamBerryiii and @suneetnangia if you want to give it a quick pass?

**Review by katriendg — APPROVED** [review]
Approving here as I've merged and need to get this to main before another merge which should include these doc changes.

**Pull request merged**